README.md

```diff
@@ -388,7 +388,34 @@ def run(nre):
     bob.send(...)
 ```
 
-> Please note that the list of accounts include only those that exist in the local `<network>.accounts.json` file.
+> Please note that the list of accounts include only those that exist in the local `<network>.accounts.json` file. In a recent release we added a flag to the command, to get predeployed accounts if the network you are connected to is a [starknet-devnet](https://github.com/Shard-Labs/starknet-devnet) instance.
+
+### `get-accounts --predeployed`
+
+This flag retrieves the predeployed accounts if the network you are connecting to is a [starknet-devnet](https://github.com/Shard-Labs/starknet-devnet) instance.
+
+You can use it either from the cli:
+
+```sh
+nile get-accounts --predeployed
+```
```

Comment:
We can't now, but it is an interesting idea. Maybe it will be worth modifying the **send** command (with a predeployed flag or similar) to allow sending tx through predeployed accounts? I'm curious about both your opinions on this @andrew-fleming @martriay

Reply:
I assumed as much, just confirming. What do you think about removing the CLI snippet and the `You can use it either from the cli`? I don't think there's any value from running the command in the CLI and it might avoid users (or me) asking if we can send txs from the predeployeds through the CLI

> Maybe it will be worth modifying the send command (with a predeployed flag or similar) to allow sending tx through predeployed accounts

Hmm it sounds like it'd be great to consider. I wouldn't include it in this PR though, that seems like a down-the-road feature

Reply:
But you can use it from the cli, just not to send transactions. You can list the predeployed accounts from the cli, as you can do with registered accounts running `nile get-accounts`, so I think thee snippet still makes sense.